dchima/goose-backend
# :tada: successfully implemented typeorm

## What it does
This pr successfully implements typeorm into the application. I used this due to being stuck trying to implement sequelize for typescript. I might try implementing that again some other time

## How to test
make sure you have your database url in the `.env` file, and simply run `npm run start:dev`. That should be all you need.

## What's next
I do need to figure out migrations before properly starting the user authentication endpoints. Right after my late evening jog
